examples/webgl_buffergeometry_instancing_dynamic.html

```diff
@@ -89,187 +88,115 @@
 
 		void main() {
 
-			gl_FragColor = texture2D(map, vUv);
+			gl_FragColor = texture2D( map, vUv );
 
 		}
-
 	</script>
 
 	<script>
 
-		if ( !Detector.webgl ) Detector.addGetWebGLMessage();
+		if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
 
 		var container, stats;
+		var camera, scene, renderer, mesh;
+		var offsetAttribute, orientationAttribute;
 
-		var camera, scene, renderer;
-		var orientations;
+		var lastTime = 0;
 
+		var moveQ = new THREE.Quaternion( 0.5, 0.5, 0.5, 0.0 ).normalize();
+		var tmpQ = new THREE.Quaternion();
+		var currentQ = new THREE.Quaternion();
+
+		init();
+		animate();
 
 		function init() {
 
 			container = document.getElementById( 'container' );
 
 			camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 1, 1000 );
-			//camera.position.z = 20;
 
 			scene = new THREE.Scene();
 			scene.background = new THREE.Color( 0x101010 );
 
-			renderer = new THREE.WebGLRenderer();
-
 			// geometry
 
 			var instances = 5000;
 
-			var geometry = new THREE.InstancedBufferGeometry();
+			var bufferGeometry = new THREE.BoxBufferGeometry( 2, 2, 2 );
 
-			// per mesh data
-			var vertices = new THREE.BufferAttribute( new Float32Array( [
-				// Front
-				-1, 1, 1,
-				1, 1, 1,
-				-1, -1, 1,
-				1, -1, 1,
-				// Back
-				1, 1, -1,
-				-1, 1, -1,
-				1, -1, -1,
-				-1, -1, -1,
-				// Left
-				-1, 1, -1,
-				-1, 1, 1,
-				-1, -1, -1,
-				-1, -1, 1,
-				// Right
-				1, 1, 1,
-				1, 1, -1,
-				1, -1, 1,
-				1, -1, -1,
-				// Top
-				-1, 1, 1,
-				1, 1, 1,
-				-1, 1, -1,
-				1, 1, -1,
-				// Bottom
-				1, -1, 1,
-				-1, -1, 1,
-				1, -1, -1,
-				-1, -1, -1
-			] ), 3 );
-
-			geometry.addAttribute( 'position', vertices );
-
-			var uvs = new THREE.BufferAttribute( new Float32Array( [
-						//x	y	z
-						// Front
-						0, 0,
-						1, 0,
-						0, 1,
-						1, 1,
-						// Back
-						1, 0,
-						0, 0,
-						1, 1,
-						0, 1,
-						// Left
-						1, 1,
-						1, 0,
-						0, 1,
-						0, 0,
-						// Right
-						1, 0,
-						1, 1,
-						0, 0,
-						0, 1,
-						// Top
-						0, 0,
-						1, 0,
-						0, 1,
-						1, 1,
-						// Bottom
-						1, 0,
-						0, 0,
-						1, 1,
-						0, 1
-			] ), 2 );
-
-			geometry.addAttribute( 'uv', uvs );
-
-			var indices = new Uint16Array( [
-				0, 1, 2,
-				2, 1, 3,
-				4, 5, 6,
-				6, 5, 7,
-				8, 9, 10,
-				10, 9, 11,
-				12, 13, 14,
-				14, 13, 15,
-				16, 17, 18,
-				18, 17, 19,
-				20, 21, 22,
-				22, 21, 23
-			] );
-
-			geometry.setIndex( new THREE.BufferAttribute( indices, 1 ) );
+			// copy data from a simple box geometry
```

Comment:
// copying data from a simple box geometry, but you can specify a custom geometry if you want

This is OK. I was a bit concerned originally because the previous code showed how to set a custom geometry -- and this PR removes that.

I think the procedure for a custom geometry can be inferred.